src/TextArea/TextArea.tsx

```diff
@@ -159,6 +159,7 @@ export class TextArea extends React.Component<Props, State> {
   render() {
     let wrapperClass = "TextArea";
     const errorMessage = this.currentErrorMessage();
+    const valid = errorMessage === "";
```

Comment:
can `errorMessage` be `null`? should we do `const valid = !errorMessage` instead?